### PR TITLE
Adds the Mississippi Queen, a new drink

### PR DESCRIPTION
**Pull request by Wallemations**
## About The Pull Request
Adds the infamous Mississippi Queen, a drink so spicy it'll make you start seeing things.
palm-export
## Why It's Good For The Game
DeficientGenuineGlowworm-size_restricted
## Changelog
:cl:
add: Bartenders can now mix up the Mississippi Queen for those willing to take the plunge.
/:cl:

### CODE REVIEW

**Comment by Wallemations** [comment]
how the hell did this fail, it ran fine when I tested it

**Comment by Wallemations** [comment]
oh my god it was the comment

File: code/modules/food_and_drinks/recipes/drinks/drinks_non-alcoholic.dm [hunk 1]
```diff
@@ -170,3 +170,7 @@
 	results = list(/datum/reagent/consumable/cucumberlemonade = 5)
 	required_reagents = list(/datum/reagent/consumable/lemon_lime = 3, /datum/reagent/consumable/cucumberjuice = 2, /datum/reagent/consumable/ice = 1)
 
+/datum/chemical_reaction/drink/mississippi_queen
+	results = list(/datum/reagent/consumable/mississippi_queen = 50)
+	required_reagents = list(/datum/reagent/consumable/tomatojuice = 15, /datum/reagent/consumable/mayonnaise = 10, /datum/reagent/consumable/soysauce = 5, /datum/reagent/consumable/vinegar = 2, /datum/reagent/consumable/capsaicin = 10, /datum/reagent/consumable/coco = 2)
```
Comment (timothymtorres) [inline code/modules/food_and_drinks/recipes/drinks/drinks_non-alcoholic.dm:175]: Not a big fan of this recipe.  Most drinks are 2-3 reagents (in rare cases 5) and the results are usually 3, 5, or 10 units.

I'd recommend using a mixed drink as one reagent ie. `hot_coco` and two other regular reagents.  Lower the result amount to be similar to other drinks.

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 2]
```diff
@@ -1664,12 +1664,12 @@
...
 /datum/reagent/consumable/mississippi_queen/on_mob_life(mob/living/carbon/drinker, delta_time, times_fired)
 	switch(current_cycle)
-		if(10 to 20) // That's called beginner's luck
+		if(10 to 20) 
 			drinker.adjust_dizzy(4 SECONDS * REM * delta_time)
 		if(20 to 30)
 			if(DT_PROB(15, delta_time))
 				drinker.adjust_confusion(4 SECONDS * REM * delta_time)
 		if(30 to 200)
-			drinker.adjust_hallucinations(1 MINUTE * REM * delta_time)
```
Comment (timothymtorres) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: You need to use `MINUTES` instead of `MINUTE`

Comment (Wallemations) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm]: RIGHT

File: code/modules/food_and_drinks/recipes/drinks/drinks_non-alcoholic.dm (hunk 1 above)
Comment (Wallemations) [reply code/modules/food_and_drinks/recipes/drinks/drinks_non-alcoholic.dm:175]: Shit, I gotta think on what to use for some of them then. I can't think of anything that would have similar reagents, and I remember I was getting yelled at for actually leaving some reagents _out_ before

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm (hunk 2 above)
Comment (Wallemations) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm]: wait it doesn't use minutes anymore, the code uses seconds

**Review by Imaginos16 — APPROVED** [review]
Funny reference, nice job.

File: code/modules/food_and_drinks/recipes/drinks/drinks_non-alcoholic.dm (hunk 1 above)
Comment (Wallemations) [reply code/modules/food_and_drinks/recipes/drinks/drinks_non-alcoholic.dm:175]: I brought it up with armhulenn and they said reducing it further would be pushing the reference further away from what it's referencing. I don't really know who's advice to take here lol

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 3]
```diff
@@ -1647,3 +1647,29 @@
...
+	switch(current_cycle)
+		if(10 to 20) 
+			drinker.adjust_dizzy(4 SECONDS * REM * delta_time)
+		if(20 to 30)
+			if(DT_PROB(15, delta_time))
+				drinker.adjust_confusion(4 SECONDS * REM * delta_time)
+		if(30 to 200)
+			drinker.adjust_hallucinations(60 SECONDS * REM * delta_time)
+
+	..()
```
Comment (Time-Green) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm:1675]:
```suggestion
	return ..()
```
This is important or shit starts breaking

**Review by Time-Green — CHANGES_REQUESTED** [review]

**Pull request merged**